.gitlab/package_build/windows.yml

```diff
@@ -33,6 +33,7 @@ windows_msi_and_bosh_zip_x64-a7:
     PYTHON_RUNTIMES: '3'
   before_script:
     - set RELEASE_VERSION $RELEASE_VERSION_7
+    - $vcpkgBlobSaSUrl = (aws ssm get-parameter --region us-east-1 --name ci.datadog-agent-buildimages.vcpkg_blob_sas_url --with-decryption --query "Parameter.Value" --out text)
```

Comment:
nit: this could be part of the script instead of the before_script, would remove one copy-paste of it.